### PR TITLE
CB-18035 NPE in datalakeWaitObject

### PR DESCRIPTION
**Pull request by yg-gd**
added test to ensure that name translation doesn't introduce any potential NPE for conversion.

See detailed description in the commit message.

### CODE REVIEW

File: datalake/src/test/java/com/sequenceiq/datalake/controller/sdx/SdxClusterConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.datalake.entity.DatalakeStatusEnum;
+import com.sequenceiq.sdx.api.model.SdxClusterStatusResponse;
+
+
+class SdxClusterConverterTest {
+
+    @Test
+    void sdxClusterStatusConverter() {
```
Comment (horadla23) [inline datalake/src/test/java/com/sequenceiq/datalake/controller/sdx/SdxClusterConverterTest.java:14]: there is no test if the input is null

**Review by horadla23 — COMMENTED** [review]
please fix checkstyle and check IT CI job

**Comment by horadla23** [comment]
the commit message is misleading, there is no datalakeWaitObject anywhere in the change

**Comment by yg-gd** [comment]
Change for DatalakeWaitObject already commited as a part of CB-18679

**Review by horadla23 — APPROVED** [review]

**Pull request merged**